Makefile

```diff
@@ -293,14 +296,15 @@ gen_build_version = sed -e s/@@GIT_SHA@@/$(git_sha)/ -e s/@@GIT_DATE_TIME@@/$(da
 # as a regular source file as part of the compilation process.
 # One can run "strings executable_filename | grep _build_" to find
 # the version of the source that we used to build the executable file.
-CLEAN_FILES += util/build_version.cc:
```

Comment:
Guessing this colon at the end was a typo as this was getting passed verbatim to `rm`. But because the file is in .gitignore, nobody noticed.